lib/parsers/asm-parser.ts

```diff
@@ -709,6 +715,19 @@ export class AsmParser extends AsmRegex implements IAsmParser {
                     labels: labelsInLine,
                 });
             }
+
+            match = line.match(this.relocationRe);
+            if (match) {
+                const address = parseInt(match.groups.address, 16);
+                const relocname = match.groups.relocname;
+                const relocdata = match.groups.relocdata;
+                // value/addend matched but not used yet.
+                const match_value = relocdata.match(this.relocDataSymNameRe);
+                asm.push({
```

Comment:
@partouf does that look the correct way to have the relocation displayed? I remember that when starting this I thought of having a dedicated structure for relocations... But then kept things simple like this.

Reply:
I think so. You could add the label/symbolname that's used, but since it probably doesn't actually exist, it wouldn't be used anyway. So I think it's good enough.

Reply:
Ok! So will replicate this in `asm-parser`